### DISCUSSION TITLE
Is the price_suggestions API covered?

### DISCUSSION THREAD

**Discussion by mmcc-xx** [joalla/discogs_client#68] [Support] [answered]
Discogs has an API under Marketplace called Price Suggestions:
https://www.discogs.com/developers#page:marketplace,header:marketplace-price-suggestions

Is that covered in this project? I've been looking through the documentation and don't see it, but I thought I'd check.

Thanks!
Clark

**Comment by AnssiAhola** [comment]
Hey Clark!

It doesn't seem to be covered, no.
If you want you can add it and submit a pull request. 😃 
If not, we can take a look at it and maybe add it in the next version or version after that, depending when's the next release.

Cheers,
Anssi

**Comment by prcutler** [comment, marked as answer]
@mmcc-xx - Update for you:  https://github.com/joalla/discogs_client/pull/71, you might want to watch that PR for when it gets merged in.

**Comment by mmcc-xx** [comment]
Thanks all!

**Reply by JOJ0** [reply]
PR #71 merged.
